chargebee_byte/client.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
+
+import requests
+
+from chargebee_byte.requests.subscription_request import SubscriptionRequest
+
+
+class Client(object):
+    def __init__(self, site, api_key):
+        self.auth = requests.auth.HTTPBasicAuth(api_key, '')
+        self.api_url = 'https://{}.chargebee.com/api/v2'.format(site)
+
+    def get_subscriptions(self, parameters=None):
+        request = SubscriptionRequest(parameters)
+        ret = requests.get(self.api_url + request.path, auth=self.auth, params=request.data)
```

Comment:
I would rename this to `response`, and add a `response.raise_for_status()` to catch 40x/50x status codes

Reply:
Good point :)